requirements.prod.txt

```diff
@@ -402,6 +402,10 @@ python3-openid==3.2.0 \
     --hash=sha256:33fbf6928f401e0b790151ed2b5290b02545e8775f982485205a066f874aaeaf \
     --hash=sha256:6626f771e0417486701e0b4daff762e7212e820ca5b29fcc0d05f6f8736dfa6b
     # via social-auth-core
+pytz==2021.3 \
```

Comment:
issue: aeb47d24203cff9bd5f9911bb392b02ffc2429dc states:

> Upgrade DRF 3.12.4 -> 3.13.1 to drop pytz dependency

But `pytz` is still here, so either the commit message is not accurate, or the requirements need rebuilding.

Reply:
Arg, they pulled it into their requirements rather than removing the dep, thanks 👍